### PR TITLE
XP tracker: Fix issue where progress bar shows 100.00% before level up

### PR DESCRIPTION
**Pull request by jdkleuver**
Saw this in discord:
2019-07-15-200239_679x101_scrot
Before:
2019-07-15-200045_227x63_scrot
After:
2019-07-15-200149_237x64_scrot

My first pull request so let me know if I did anything wrong!

### CODE REVIEW

**Comment by deathbeam** [comment]
Should this be really solved like this? this looks like rounding issue, so maybe avoid rounding

**Comment by abextm** [comment]
Probably could just configure the DecimalFormat to round down

**Comment by jdkleuver** [comment]
I took your feedback into account, see new commit

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java [hunk 1]
```diff
@@ -226,6 +227,7 @@ private void rebuildAsync(boolean updated, boolean skillPaused, XpSnapshotSingle
 
 			// Update progress bar
 			progressBar.setValue((int) xpSnapshotSingle.getSkillProgressToGoal());
+			TWO_DECIMAL_FORMAT.setRoundingMode(RoundingMode.DOWN);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java]: this should be in static { } constructor, e.g

```java
static
{
    			TWO_DECIMAL_FORMAT.setRoundingMode(RoundingMode.DOWN);
}
```

**Comment by jdkleuver** [comment]
@deathbeam is that change ok? tbh I've never used a static block like that before so thanks for bringing it to my attention

**Comment by deathbeam** [comment]
Yeah, looks fine

**Pull request merged**